jenkins/Dockerfile

```diff
@@ -1,12 +1,15 @@
-FROM docker.io/jenkins/jenkins:2.462.3-jdk17@sha256:429647d4688daa3ca2520fb771a391bae8efa1e4def824b32345f13dde223227
+FROM docker.io/jenkins/jenkins:2.462.2-jdk17
```

Comment:
why downgrade?
this is the latest lts version that is also used in the chart.
https://artifacthub.io/packages/helm/jenkinsci/jenkins